scripts/templates/installer.sh

```diff
@@ -1171,10 +1201,14 @@ exxit() {
     cp -f "$gapps_removal_list" "$TMP/logs/gapps-remove_revised.txt"
     cp -f "$rec_cache_log" "$TMP/logs/Recovery_cache.log"
     cp -f "$rec_tmp_log" "$TMP/logs/Recovery_tmp.log"
-    curLD="$LD_LIBRARY_PATH"
-    unset LD_LIBRARY_PATH
+    OLD_LD_PATH=$LD_LIBRARY_PATH
+    OLD_LD_PRE=$LD_PRELOAD
+    OLD_LD_CFG=$LD_CONFIG_FILE
+    unset LD_LIBRARY_PATH LD_PRELOAD LD_CONFIG_FILE
     logcat -d -f "$TMP/logs/logcat"
-    export LD_LIBRARY_PATH="$curLD"
+    test "$OLD_LD_PATH" && export LD_LIBRARY_PATH=$OLD_LD_PATH
```

Comment:
I just want to mention that  [ expression ] is the same as `test` in Unix.

I assume that quite some code is extracted from your other project, so I am not going to complain about any code-style, because that would only increase any maintenance burden. Best is code to keep shared code as alike as possible for maintainability.

Reply:
My idea would be to use shellformat, aka shfmt

Reply:
Yep, I'm aware those built-ins are aliases, but you're right, the idea was to leave it matching the source for upkeep reasons, just as it was on the previous merges for it. I'll consider updating this style tic of mine in the AK3 source in the future so we can all be happy perhaps. 😛

As for shfmt, there is an old Issue, #48, from @mfonville that suggested using ShellCheck linter, if someone was finally wants to pull the trigger on that, which would be cool for sure. 🙂

Reply:
Updated AK3 and the PR to remove all use of test, so this should be good now.